Dockerfile

```diff
@@ -11,9 +11,14 @@ RUN dnf install -y \
         rubygem-asciidoctor && \
     dnf groupinstall -y development-tools && \
     gem install \
+        asciidoctor-tabs:1.0.0.beta.6 \
+        bundler:2.3.27 \
```

Comment:
Would it be better to the package `rubygem-bundler`?

Reply:
Tested on both "master" and PR 3104 with tabs; works. -> Applied.